pkg/kubelet/cm/deviceplugin/device_plugin_stub.go

```diff
@@ -48,9 +54,18 @@ func NewDevicePluginStub(devs []*pluginapi.Device, socket string) *Stub {
 
 		stop:   make(chan interface{}),
 		update: make(chan []*pluginapi.Device),
+
+		allocFunc: nil,
 	}
 }
 
+// SetAllocFunc sets allocFunc of the device plugin
+func (m *Stub) SetAllocFunc(f stubAllocFunc) error {
+	m.allocFunc = f
+
+	return nil
```

Comment:
If we are not returning any special error, maybe simply skip return variable in this func?

Reply:
Sure.